examples/server.rs

```diff
@@ -0,0 +1,158 @@
+use prosa_utils::config::tracing::TelemetryFilter;
+use prosa_utils::msg::simple_string_tvf::SimpleStringTvf;
+use serde::{Deserialize, Serialize};
+use tracing::debug;
+
+/// Demo Hyper processor adaptor
+#[derive(Debug, Adaptor, Clone)]
+pub struct HyperDemoAdaptor {
+    prosa_name: String,
+}
+
+impl<M> HyperServerAdaptor<M> for HyperDemoAdaptor
+where
+    M: 'static
+        + std::marker::Send
+        + std::marker::Sync
+        + std::marker::Sized
+        + std::clone::Clone
+        + std::fmt::Debug
+        + prosa_utils::msg::tvf::Tvf
+        + std::default::Default,
+{
+    fn new(
+        _proc: &HyperServerProc<M>,
+        prosa_name: &str,
+    ) -> Result<Self, Box<dyn ProcError + Send + Sync>> {
+        Ok(HyperDemoAdaptor {
+            prosa_name: prosa_name.into(),
+        })
+    }
+
+    async fn process_http_request(
+        &self,
+        req: Request<hyper::body::Incoming>,
+        h2: bool,
+    ) -> crate::HyperResp<M> {
+        match req.uri().path() {
+            "/" => HyperResp::HttpResp(
+                Response::builder()
+                    .body(BoxBody::new(Full::new(Bytes::from(format!(
+                        "{} - Home of {}",
+                        if h2 { "H2" } else { "HTTP/1.1" },
+                        self.prosa_name,
+                    )))))
+                    .unwrap(),
+            ),
+            "/test" => {
+                let mut tvf_req = M::default();
+                tvf_req.put_string(1, req.method().to_string());
+                tvf_req.put_string(2, "/test");
+                HyperResp::SrvReq(String::from("SRV_TEST"), tvf_req)
+            }
+            _ => HyperResp::HttpResp(
+                Response::builder()
+                    .status(404)
+                    .body(BoxBody::new(Full::new(Bytes::from("Not Found"))))
+                    .unwrap(),
+            ),
+        }
+    }
+
+    fn process_srv_response(&self, resp: &M) -> Response<BoxBody<Bytes, Infallible>> {
+        let body = resp
+            .get_string(10)
+            .unwrap_or(Cow::Owned(String::from("empty body")));
+        Response::builder()
+            .body(BoxBody::new(Full::new(Bytes::from(format!(
+                "Body: {body}\nTvfResp: {resp:?}"
+            )))))
+            .unwrap()
+    }
+}
+
+#[settings]
+#[derive(Debug, Default, Deserialize, Serialize, Clone)]
+pub(crate) struct MainHyperSettings {
+    pub(crate) hyper: HyperServerSettings,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let matches = Command::new("hyper")
+        .version(env!("CARGO_PKG_VERSION"))
+        .author(env!("CARGO_PKG_AUTHORS"))
+        .about(env!("CARGO_PKG_DESCRIPTION"))
+        .arg(arg!(-v - -verbose).action(ArgAction::SetTrue))
+        .arg(arg!(-s --stub "Start a Stub processor").action(ArgAction::SetTrue))
+        .arg(
+            arg!(-c --config <CONFIG_PATH> "Path of the JPTR ProSA configuration file")
```

Comment:
JPTR SOULD not be used here.

```suggestion
            arg!(-c --config <CONFIG_PATH> "Path of the HYPER ProSA configuration file")
```

Reply:
Corrected. Miss this one...